rsmtool/reader.py

```diff
@@ -11,17 +11,64 @@
 """
 
 import warnings
-import pandas as pd
 
 from functools import partial
 from os.path import (abspath,
                      exists,
                      join,
                      splitext)
 
+import pandas as pd
+
 from rsmtool.container import DataContainer
 
 
+
+def read_jsonlines(filename, converters=None):
+    """
+    Read jsonlines from a file.
+    Normalize nested jsons with up to one level of nesting
+
+    Parameters
+    ----------
+    filename: str
+        Name of file to read
+    converters : dict or None, optional
+        A dictionary specifying how the types of the columns
+        in the file should be converted. Specified in the same
+        format as for `pd.read_csv()`.
+
+    Returns
+    -------
+    df : pandas DataFrame
+         Data frame containing the data in the given file.
+    """
+
+    df = pd.read_json(filename,
+                      orient='records',
+                      lines=True,
+                      dtype=converters)
+
+    # make sure we didn't get a plain json
+    if type(df.columns) == pd.RangeIndex:
+        raise ValueError("It looks like {} is a simple json file. "
```

Comment:
Hmm, how do we distinguish between a "simple json" file and a jsonlines file with a single instance?

Reply:
Got it. Thanks!